internal/server/middlewares/cors.go

```diff
@@ -0,0 +1,24 @@
+// internal/server/middleware/cors.go
+package middlewares
+
+import (
+    "net/http"
+)
+
+// CORSMiddleware allows all origins and sets necessary headers
+func CORSMiddleware(next http.Handler) http.Handler {
+    return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+        // Allow all origins
+        w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
here, it is not taken from configs

Reply:
That's true, I rethought it backthen. Should have removed it.